core/src/main/java/hudson/model/Run.java

```diff
@@ -1,6 +1,8 @@
 /*
  * The MIT License
- * 
+ *
+ * Copyright (c) 2019 Intel Corporation 
```

Comment:
It is better to modify the copyright below and to just add Intel to the list

Reply:
Okay, I will adjust it appropriately.
This code was lifted from internal code changes, where the policy is that the newest copyright has to go on top. :)

As for the implementation as an extension:  We considered the same, and created a larger change, which also enables Jenkins to compress the logs on its own at the end of the build.

If you prefer, we can upload that larger change, instead of this smaller change.
Our assumption was just that this smaller change might be easier to get upstreamed first, before we come in with the larger alteration.